packages/core/src/auth/deprecated/loginManager.ts

```diff
@@ -181,7 +181,7 @@ export async function loginWithMostRecentCredentials(
             }
             getLogger().info('autoconnect: connected: %O', asString(creds))
             if (popup) {
-                await vscode.window.showInformationMessage(
+                void vscode.window.showInformationMessage(
```

Comment:
Yeah I don't think it can fail, at least I've never seen a case where it would. I briefly skimmed the code on vscode repo and couldn't find any failure cases.
But even it if fails to show I don't think it's expected that that we also throw in these functions.